content/en/docs/reference/access-authn-authz/rbac.md

```diff
@@ -1125,7 +1125,7 @@ Examples:
 
 Default RBAC policies grant scoped permissions to control-plane components, nodes,
 and controllers, but grant *no permissions* to service accounts outside the `kube-system` namespace
-(beyond discovery permissions given to all authenticated users).
+(beyond the permissions given by [API discovery roles](#discovery-roles)).
```

Comment:
Service accounts also have access to API discovery and soon will have access to cluster trust bundles.  There are probably other things I am forgetting.

Reply:
Hi @enj,
Do you wants to hold this PR till the time Service accounts get access to cluster trust bundles and other if any?

Reply:
Hi @enj,
Just pinging as a reminder for above,
Thanks.

Reply:
the linked section includes all the discovery-style roles granted to all authenticated users... I would assume any future roles like that would be included there

/lgtm
/approve